tests/Cache/CacheClientTest.php

```diff
@@ -447,6 +447,88 @@ public function testGetTimeout()
         $this->assertEquals(MomentoErrorCode::TIMEOUT_ERROR, $response->asError()->errorCode());
     }
 
+    // SetIfNotExists tests
+
+    public function testSetIfNotExists()
+    {
+        $key = uniqid();
+        $value = uniqid();
+
+        $response = $this->client->setIfNotExists($this->TEST_CACHE_NAME, $key, $value);
+        $this->assertNull($response->asError());
+        $response = $response->asStored();
```

Comment:
Do you want to an assertion about `$response` not being null here? Also, if you expect this to *not* return `NotStored` you might assert that `asNotStored` returns null.

Reply:
Whoops, thanks for catching the missing assertion!

Reply:
Done in [374d3ff](https://github.com/momentohq/client-sdk-php/pull/91/commits/374d3ff8477a745d539875da8e48bad5c2846c88)